tests/utils.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func RunOnNode(node string, command string) (string, error) {
+	provider, ok := os.LookupEnv("KUBEVIRT_PROVIDER")
+	if !ok {
+		panic("KUBEVIRT_PROVIDER environment variable must be specified")
+	}
+
+	out, err := exec.Command("docker", "exec", provider+"-"+node, "ssh.sh", command).CombinedOutput()
```

Comment:
I am not so comfortable with this function.
Can we just use a job with affinity to a specific node?

Reply:
Correct me if I'm wrong, but that would require us to build a special image for this job, mount ovs socket, communicate with the image using client go.

Can I just add a TODO? I feel like that's out of scope of this patch.

Reply:
Sure you can add a TODO and we will come back to it after this PR gets merge